components/schemes/State/examples/src/MDCColorResource.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+// Note: Make sure to update [semanticResourceFromName:] when changing or adding values
+typedef NS_ENUM(NSInteger, MDCColorResourceSemantic) {
+  MDCColorResourceSemanticNone,
```

Comment:
Can you add comments to these types? Of note, I don't understand what a `.None` Semantic represents.